Common/include/option_structure.hpp

```diff
@@ -952,19 +950,59 @@ inline TURB_FAMILY TurbModelFamily(TURB_MODEL model) {
     case TURB_MODEL::SA_E_COMP:
       return TURB_FAMILY::SA;
     case TURB_MODEL::SST:
-    case TURB_MODEL::SST_SUST:
       return TURB_FAMILY::KW;
   }
   return TURB_FAMILY::NONE;
 }
 
+/*!
+ * \brief SST Options
+ */
+enum class SST_OPTIONS {
+  NONE,        /*!< \brief No SST Turb model. */
+  V1994,       /*!< \brief 1994 Menter k-w SST model. */
+  V2003,       /*!< \brief 2003 Menter k-w SST model. */
+  MODIFIED,    /*!< \brief Modified k-w SST model. */
+  SUST,        /*!< \brief Menter k-w SST model with sustaining terms. */
+  VORTICITY,   /*!< \brief Menter k-w SST model with vorticity production terms. */
+  KL,          /*!< \brief Menter k-w SST model with Kato-Launder production terms. */
+  UNCERTAINTY, /*!< \brief Menter k-w SST model with uncertainty quantification modifications. */
+  RC,          /*!< \brief Menter k-w SST model with rotation/curvature corrections. */
+};
+static const MapType<std::string, SST_OPTIONS> SST_Options_Map = {
+  MakePair("NONE", SST_OPTIONS::NONE)
+  MakePair("V1994", SST_OPTIONS::V1994)
+  MakePair("V2003", SST_OPTIONS::V2003)
+  MakePair("MODIFIED", SST_OPTIONS::MODIFIED)
```

Comment:
For fast integration I think the way forward is:
- Keep the inconsistent 1994 as the default.
- Allow 2003 as option but do not expose MODIFIED, we will use modified by default, and in the future we can add `NOT_MODIFIED` to the options, this saves you the trouble of implementing the changes needed to the stress tensor. Moreover we should have robust default options, and the -m versions are better for that, probably.
- The 2003m would of course not feature the inconsistency from #1551, and we can issue a warning when defaulting to 1994 that users should prefer 2003 which will eventually become the default.

If having "NOT_MODIFIED" in the future does not sound good and alternative is to have V1994m V2003m now, and then add V1994 and V2003.
```suggestion
  MakePair("V1994", SST_OPTIONS::V1994)
  MakePair("V2003", SST_OPTIONS::V2003)
```

Reply:
Only inconsistent 1994m and proper 2003m, for now.
We warn that users should change to 2003m and that we will make it the default (release notes, CFD-online, we can tweet it as well...).
Then once we have the opportunity to make breaking changes we **fix** 1994m and we make 2003m the default.
If/when proper 1994 / 2003 are implemented does not matter since they'll be new, and for that reason they also don't need to be tackled now.

Reply:
sounds good, and a reasonable work package